nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,584 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""Provides interfaces to various commands for running PET analyses provided by FreeSurfer
+"""
+
+import os
+
+from ... import logging
+from ..base import (
+    TraitedSpec,
+    File,
+    traits,
+    Directory,
+    InputMultiPath,
+    isdefined,
+)
+from .base import FSCommand, FSTraitedSpec
+
+from .model import GLMFitInputSpec, GLMFit
+
+__docformat__ = "restructuredtext"
+iflogger = logging.getLogger("nipype.interface")
+
+
+class GTMSegInputSpec(FSTraitedSpec):
+
+    subject_id = traits.String(argstr="--s %s", desc="subject id", mandatory=True)
+
+    xcerseg = traits.Bool(
+        argstr="--xcerseg",
+        desc="run xcerebralseg on this subject to create apas+head.mgz",
+    )
+
+    out_file = File(
+        "gtmseg.mgz",
+        argstr="--o %s",
+        desc="output volume relative to subject/mri",
+        usedefault=True,
+    )
+
+    upsampling_factor = traits.Int(
+        argstr="--usf %i", desc="upsampling factor (default is 2)"
+    )
+
+    subsegwm = traits.Bool(
+        argstr="--subsegwm", desc="subsegment WM into lobes (default)"
```

Comment:
should this default to `True`?

Reply:
Hmm. Thanks for pointing this out. According to https://github.com/freesurfer/freesurfer/blob/dev/mri_gtmseg/mri_gtmseg.cpp this should be --subseg-wm and default is false? This is not in line with the documentation of the binary I have at hand (v. 7.2).